Utils/Dataflow/030_PDFAnalyzer/manager.py

```diff
@@ -1826,23 +1826,27 @@ def export_all(self, quick=False, n=None, n_p=None, errors=None, attr=None,
                     else:
                         s += ","
                     other = ""
-                    tables = ""
+                    tables = []
                     for c in outp["content"]:
                         if not other and c.endswith("datasets") and\
                            not c.startswith("mc") and not c.startswith("real"):
-                            attr["other_datasets"].append(p.fname)
                             other = "1"
-                        if not tables and c.startswith("table"):
-                            attr["dataset_tables"].append(p.fname)
-                            tables = "1"
+                        if c.startswith("table"):
+                            tables.append(int(c[len("table_"):]))
+                    if other:
+                        attr["other_datasets"].append(p.fname)
+                    tables.sort()
+                    tables = str(tables).replace(",", "").strip("[]")
+                    if tables:
+                        attr["dataset_tables"].append(p.fname)
                     s += "%s,%s," % (other, tables)
                     for a in Paper.attributes_general:
                         if outp["content"]["plain_text"][a]:
                             attr[a].append(p.fname)
-                            if a == "atlas_name":
-                                s += "%s," % outp["content"]["plain_text"][a]
-                            else:
-                                s += "1,"
+                            add = outp["content"]["plain_text"][a]
+                            if a == "campaigns":
```

Comment:
Is it only for "campaigns", or for any `list`?
Maybe use `if isinstance(a, list)` or `if type(a) is list`?
Second variant is more readable, of course, but less universal (it can be another class, inheriting from `list` -- though it is not so often in our code).